sympy/assumptions/refine.py

```diff
@@ -70,6 +70,9 @@ def refine_abs(expr, assumptions):
         return arg
     if ask(Q.negative(arg), assumptions):
         return -arg
+    # arg is Mul
+    if isinstance(arg, Mul):
+        return refine(abs(arg.args[0]), assumptions) * refine(abs(arg / arg.args[0]), assumptions)
```

Comment:
Might we keep those that don't simplify out of Abs *in* the Abs? e.g. `Abs(x, y, z) -> x * Abs(y*z)` if only x is positive?

```
If isinstance(arg, Mul):
    r = [refine(abs(a), assumptions) for a in arg.args]
    nonabs = []
    inabs = []
    for i in r:
        if isinstance(i, Abs):
            inabs.append(i.args[0])
        else:
            nonabs.append(i)
    return Mul(*nonabs)*Abs(Mul(*inabs))
```